dashboard/app/models/stage.rb

```diff
@@ -67,7 +67,11 @@ def localized_name
   end
 
   def localized_category
-    I18n.t "flex_category.#{flex_category}" if flex_category
+    if flex_category
+      I18n.t "flex_category.#{flex_category}"
+    else
+      I18n.t "flex_category.content"
+    end
```

Comment:
This is a localized string for a stages category.

Previously, in cases where a category wasn't explicitly defined, the server would pass down nothing for this, and let the client decide how to handle it (This happened eventually at the view layer in a non-localized manner https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/code-studio/components/progress/course_progress.jsx#L65).

My approach is to have the server always pass this down, such that we always have a flex_category (in most cases it will just be Content).